src/hooks/useMapToReactNumberConfig.ts

```diff
@@ -0,0 +1,32 @@
+import { useAppSelector } from 'src/hooks/useAppSelector';
+import { appLanguageStateSelector } from 'src/selectors/appLanguageStateSelector';
+import { formatNumber } from 'src/utils/formattingUtils';
+import type { IInputFormatting } from 'src/layout/layout';
+import type { CurrencyFormattingOptions, UnitFormattingOptions } from 'src/utils/formattingUtils';
+
+export const useMapToReactNumberConfig = (value: string | undefined, formatting: IInputFormatting) => {
+  const appLanguage = useAppSelector(appLanguageStateSelector);
+
+  if ((!formatting?.currency && !formatting?.unit) || !value) {
```

Comment:
I think we can remove the !value from the if statement by setting an empty string as a default value to the value argument. 😄


```suggestion
export const useMapToReactNumberConfig = (value = '', formatting: IInputFormatting) => {
  const appLanguage = useAppSelector(appLanguageStateSelector);

  if (!formatting?.currency && !formatting?.unit) {
```